### ISSUE TITLE
List index out of range

### ISSUE THREAD

**Issue by kcoderhtml** [Owen-Cochell/PaperMC-Update#13]
I love this tool and running it today found this error:
```
Traceback (most recent call last):
  File "server_update.py", line 1740, in <module>
    serv.get_new(default_version=args.version, default_build=args.build, backup=not (args.no_backup or args.new),
  File "server_update.py", line 1385, in get_new
    ver, build = self.version_select(default_version=default_version, default_build=default_build)
  File "server_update.py", line 1336, in version_select
    build = self._select('', nums, new_default, "build")
  File "server_update.py", line 1555, in _select
    output("# Selecting latest {} - [{}] ...".format(name, choice[-1]))
IndexError: list index out of range

```

**Comment by Owen-Cochell** [comment]
Thanks for pointing this out!

As of now, Paper currently has no builds available for version 1.19, even though it is listed on the API version list. This confuses the script and causes it to fail as you described if you use the 'latest' keyword (which is the default), or hard set the version to 1.19. I am pushing a fix for this in the next hour or two. If this problem is encountered after this fix, then a message will be printed to the terminal which explains the issue, and gracefully exits. This problem will eventually solve itself when Paper creates builds for 1.19.

The only thing I can recommend right now is hard setting your game version to something older (like 1.18). As of now, and after the fix, your install won't be affected when this problem occurs, so running the script will do no damage. The only thing you miss out on is new builds for older versions if you have the version set to 'latest'.

Thanks again!

**Comment by kcoderhtml** [comment]
Thanks. I absolutely love this tool it saves me so much time!

**Comment by Owen-Cochell** [comment]
Thank you for the support! This issue should be officially fixed now, the version you want is 2.2.1. If that solves your problem then please feel free to close this issue.

Thanks again!

**Comment by kcoderhtml** [comment]
Thanks!
